src/openseadragon.js

```diff
@@ -190,6 +190,16 @@
   *     Zoom level to use when image is first opened or the home button is clicked.
   *     If 0, adjusts to fit viewer.
   *
+  * @property {String|DrawerImplementation|Array} [drawer = ['webgl', 'context2d', 'html']]
```

Comment:
I think I prefer using the term `canvas` over `context2d`. I know that WebGL is technically also canvas-based, but I think in general in the industry the term people use to refer to the 2D canvas is "canvas", and that's certainly the term we've been using here in OSD.

Reply:
To me `canvas` should refer to the HTML element and the underlying rendering contexts should be called by their own names, but who am I to argue with a whole industry? :) I'll change everything to `canvas` and related.

Reply:
😂 Right? I agree it makes sense to think in terms of the elements, but yeah, let's not fight the industry on this!